recipe/meta.yaml

```diff
@@ -24,14 +24,14 @@ requirements:
         - python
         - setuptools
         - numpy
-        - cdat-lite
-        - udunits2
+        - netCDF4
+        - cfunits
```

Comment:
These two should both exist on conda-forge:
https://github.com/conda-forge/netcdf4-feedstock/
https://github.com/conda-forge/cfunits-feedstock/

@ocefpaf: I'm hoping it will pass the test now. Thanks